src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import {
+  Container,
+  TitleContainer,
+  Title,
+  CloseButton,
+  Content,
+  Fade,
+} from './Modal.style';
+
+export interface ModalProps {
+  isOpen: boolean;
+  hide: () => void;
+  modalContent: JSX.Element;
+  TitleText: string;
+}
+
+export const Modal = ({
+  isOpen,
+  hide,
+  modalContent,
```

Comment:
I think this can just be the `children`. So from the parent component, you do
```
<Modal>
    <div>stuff</div>
</Modal>
```
and this will just be `children` prop,
```
<Content>
    {children}
</Content>
```